website/project/tasks.py

```diff
@@ -49,3 +51,19 @@ def on_node_updated(node_id, user_id, first_save, saved_fields, request_headers=
                     }]
                 },
             }, headers={'Authorization': 'Bearer {}'.format(settings.SHARE_API_TOKEN)}).raise_for_status()
+
+
+@celery_app.task(ignore_results=True)
+def institution_dashboard_display(inst):
```

Comment:
Minor, but let's call this `institution_set_dashboard_display()` to keep it from looking like the property.